<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>wpisz liczbÄ™</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zgadywanka Liczbowa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Zgaduj!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wynik funkcji:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wynik">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>wynik</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>294</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunkcja">
    <property name="title">
     <string>Funkcje</string>
    </property>
    <addaction name="actionLiniowa"/>
    <addaction name="actionKwadratowa"/>
   </widget>
   <addaction name="menuFunkcja"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLiniowa">
   <property name="text">
    <string>Liniowa</string>
   </property>
  </action>
  <action name="actionKwadratowa">
   <property name="text">
    <string>Kwadratowa</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
